lblod/frontend-gelinkt-notuleren
# dev-tooling: add support for `dotenv` when in environment.js file

### Overview
Add support for loading environment variables from a `.env` file in `environment.js`. A `.env` file is only loaded when `environment` is 'development'.

The following variables are supported:
- `MOW_REGISTRY_SPARQL_ENDPOINT`
- `PUBLICATION_SPARQL_ENDPOINT`
- `ROADSIGN_IMAGE_BASE_URL`
- `FALLBACK_CODELIST_SPARQL_ENDPOINT`
- `REGULATORY_STATEMENT_ENDPOINT`

This ensures that a developer can easily override some variables inside a `.env` file (which is part of `.gitignore`), without needing to adjust the `environment.js` (and accidentally committing it).

##### connected issues and PRs:
None

### How to test/reproduce
- Try to add some of the above variables to your `.env` file and ensure the application uses them correctly.
